File: twoSum.py
from curses import keyname


def twoSum(nums, target):
    dict = {}
    array = []
    for i in range(len(nums)):
        toFind = target - nums[i]
        if toFind in dict.keys():
            array.append(dict[toFind])
            array.append(i) 
            return array
        else:
            dict[nums[i]] = i
        
    
    







nums = [3,2,4]
target = 6
print(twoSum(nums,target))

File: twoSum.js
var twoSum = function(nums, target) {
    var ans = [];
    var dict = {};
    for(var i = 0; i < nums.length; i++){
        var toFind = target - nums[i];
        if(toFind in dict){
            ans.push(i);
            ans.push(dict[toFind]);
        }
        else{
            dict[nums[i]] = i;
        }
    }
    return ans;
};
